templates/predictions.html

```diff
@@ -214,38 +253,38 @@ <h3 class="nba-standings-sub-header">WESTERN</h3>
                 <div class="table-responsive">
                     <table class="table table-bordered">
                         <thead style="border-color: white;">
-                        <tr>
-                            <th class="col-md-1">TEAM NAME</th>
-                            <th class="col-md-2">WINS</th>
-                            <th class="col-md-3">LOSES</th>
-                        </tr>
+                            <tr>
```

Comment:
@iamalexk the backend code seems to be working well! However there seems to now be a bug in how the standings are being displayed.

Before:
![Screen Shot 2020-12-11 at 2 19 16 PM](https://user-images.githubusercontent.com/18124349/101960414-39877080-3bbc-11eb-90f3-ecfe44931ed1.png)

After:
![Screen Shot 2020-12-11 at 2 19 27 PM](https://user-images.githubusercontent.com/18124349/101960428-41dfab80-3bbc-11eb-83d7-7dbeb35e6fea.png)

Do you know what may have caused this change?

Reply:
Not quite sure. I don't think it was caused by any changes I made since the only changes I made in predictions.html were what was inside the <th> tags. The attributes inside the <th> tags are even the same (col-md-1, col-md-2, etc). When I run my predictions page, I am unable to see the screenshot you have in the "before" section. My page looks as following 
<img width="1043" alt="Screen Shot 2020-12-11 at 5 37 52 PM" src="https://user-images.githubusercontent.com/47584707/101961602-0e127f00-3bd8-11eb-9861-05360d33a94e.png">
<img width="1024" alt="Screen Shot 2020-12-11 at 5 38 23 PM" src="https://user-images.githubusercontent.com/47584707/101961611-136fc980-3bd8-11eb-8c23-6dba4a32d2e5.png">
I think the remote repo is currently set up to look like the "after" screenshots because whether I'm on my issue branch or master branch, it looks the same either way.

Reply:
You're absolutely right, seems it is an issue with the master branch itself. The "after" image is only a problem in the Firefox browser, but seems to be working fine on Chrome. I'll open a separate issue regarding this 👍